tests/ES.FX.Serilog.Tests/Lifetime/ProgramEntrySerilogExtensionsTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿using ES.FX.Hosting.Lifetime;
+using ES.FX.Serilog.Lifetime;
+using Microsoft.Extensions.Hosting;
+using Moq;
+using Serilog;
+using Serilog.Events;
+
+namespace ES.FX.Serilog.Tests.Lifetime
+{
+    public class ProgramEntrySerilogExtensionsTests
```

Comment:
Needs more tests to check if serilog is actually added as the logger. (from a functional pov)